### PR TITLE
Release btc and fixes test cases of gas estimation

### PR DESCRIPTION
**Pull request by pamgonzalez**

### CODE REVIEW

File: rskj-core/src/test/java/co/rsk/peg/performance/ReceiveHeaderTest.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+                Helper.getRandomHeightProvider(10),
+                stats,
+                (environment, executionResult) -> {
+                    int result = new BigInteger(executionResult).intValue();
+                    Assert.assertEquals(-4, result);
+                }
+        );
+    }
+
+    private ABIEncoder getABIEncoder(boolean testExistedBlock) {
```
Comment (josedahlquist) [inline rskj-core/src/test/java/co/rsk/peg/performance/ReceiveHeaderTest.java:106]:
```suggestion
    private ABIEncoder getABIEncoder(boolean testExistingBlock) {
```

File: rskj-core/src/test/java/co/rsk/peg/performance/ReleaseBtcTest.java [hunk 2]
```diff
@@ -33,45 +33,115 @@
...
             BigInteger weis = Denomination.satoshisToWeis(BigInteger.valueOf(satoshis));
             ECKey sender = new ECKey();
 
             return Helper.buildSendValueTx(sender, weis);
         };
 
-        ExecutionStats stats = new ExecutionStats("releaseBtc");
-        executeAndAverage("releaseBtc", 1000, abiEncoder, storageInitializer, txBuilder, Helper.getRandomHeightProvider(10), stats);
+        executeAndAverage(
+                "releaseBtc_success",
```
Comment (josedahlquist) [inline rskj-core/src/test/java/co/rsk/peg/performance/ReleaseBtcTest.java:104]:
```suggestion
                "releaseBtc_refund",
```

**Review by josedahlquist — APPROVED** [review]

**Review by josedahlquist — DISMISSED** [review]

**Review by josedahlquist — DISMISSED** [review]

**Comment by josedahlquist** [comment]
pipeline:run

**Review by josedahlquist — APPROVED** [review]

**Comment by pamgonzalez** [comment]
Review PR: https://github.com/rsksmart/rskj/pull/1487

**Pull request closed without merging**